radixdlt-java/src/test/java/com/radixdlt/client/core/atoms/MessageParticleTest.java

```diff
@@ -16,6 +16,7 @@ public void testApplicationMetaData() {
 		MessageParticle messageParticle = new MessageParticleBuilder()
 			.payload(new byte[0])
 			.from(mock(RadixAddress.class))
+			.to(mock(RadixAddress.class))
```

Comment:
Why did this change?

Reply:
Because the unit test was broken when I tried to run them